### PR TITLE
feat: update runtime package to 4.9.2

### PR DESCRIPTION
**Pull request by luhan2017**
## Description

update runtime package to 4.9.2.
no schema update in this patch and all samples are good. 

## Task Item

#minor

### CODE REVIEW

File: runtime/dotnet/azurewebapp/Microsoft.BotFramework.Composer.WebApp.csproj [hunk 1]
```diff
@@ -16,18 +16,18 @@
...
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.ApplicationInsights.Core" Version="4.9.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.9.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.9.1" />
-    <PackageReference Include="Microsoft.Bot.Connector" Version="4.9.1" />
+    <PackageReference Include="Microsoft.Bot.Builder" Version="4.9.2" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.9.2" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.9.2" />
+    <PackageReference Include="Microsoft.Bot.Builder.ApplicationInsights" Version="4.9.2" />
+    <PackageReference Include="Microsoft.Bot.Builder.Azure" Version="4.9.2" />
+	<PackageReference Include="Microsoft.Bot.Builder.Dialogs.Declarative" Version="4.9.2" />
```
Comment (carlosscastro) [inline runtime/dotnet/azurewebapp/Microsoft.BotFramework.Composer.WebApp.csproj:24]: missing space

Comment (carlosscastro) [reply runtime/dotnet/azurewebapp/Microsoft.BotFramework.Composer.WebApp.csproj:24]: Or extra space depending on the editor :)

**Comment by boydc2014** [comment]
@lei9444  can you update the bf-cli dependencies to a release version.

File: runtime/dotnet/azurewebapp/Microsoft.BotFramework.Composer.WebApp.csproj (hunk 1 above)
Comment (luhan2017) [reply runtime/dotnet/azurewebapp/Microsoft.BotFramework.Composer.WebApp.csproj]: Good catch, fixed :)

**Comment by lei9444** [comment]
update in #3217

**Pull request merged**